frontend/src/views/FindCarSpecificationView.js

```diff
@@ -26,19 +26,47 @@ const useStyles = makeStyles((theme) => ({
 
 const specs = [
```

Comment:
**const specs** 

This is the whole array.

You copy this as is, and put it in a new file. Since you want to be able to import this, you need to export 
so sth like: **export const specs**

then in this file you add import statement: import {specs} from "./NewFile.js"